packages/@aws-cdk/assets/lib/staging.ts

```diff
@@ -53,6 +58,7 @@ export class Staging extends Construct {
     super(scope, id);
 
     this.sourcePath = props.sourcePath;
+    this.sourceHash = fingerprint(this.sourcePath);
```

Comment:
Shouldn't we pass in any options here?

Reply:
There is nothing to pass... The `Staging` class only gets a `sourcePath` and nothing else.

Reply:
I think we should expose the copy options (exclude, symlinks), but this can be in a subsequent PR.